v5.0.5/README.md

```diff
@@ -2,12 +2,22 @@
 
 ## Versions
 
-- Cosmos Hub Mainnet `cosmoshub-4` currently uses [gaia v5.0.5](https://github.com/cosmos/gaia/releases/tag/v5.0.5).
-- [Upgrade `gaiad` version with Gravity DEX that is rebased to Cosmos SDK v0.43.0](https://github.com/b-harvest/gravity-dex/blob/upgrade-liquidity-module-based-sdk-43)
+- Cosmos Hub Mainnet `cosmoshub-4` version: [gaia v5.0.5](https://github.com/cosmos/gaia/releases/tag/v5.0.5)
+- Forked version of `gaiad`: [the version with Gravity DEX that is based on Cosmos SDK v0.43.0](https://github.com/b-harvest/gravity-dex/tree/upgrade-liquidity-module-based-sdk-43)
```

Comment:
```suggestion
- Forked version of `gaiad`: [the version with Gravity DEX that is based on Cosmos SDK v0.44.0](https://github.com/b-harvest/gravity-dex/tree/upgrade-liquidity-module-based-sdk-43)
```

Reply:
Test is still ongoing, but I will update this now with some other areas where v0.43.0 is used.